app/src/main/java/com/example/mytrainingpal/util/HelperClasses.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.mytrainingpal.util
+
+class ExerciseDetails(var sets: Int,var reps: Int,var weight: Int) {}
+
+// used to pass ints by reference
+class IntHolder(var value: Int) {}
```

Comment:
Could you please move `TrainingStats` here? This is also a helper data class.
 See: https://github.com/MichaelBuessemeyer/MyTrainingsPal/blob/5512f0740ea8362e7d391ac8d27fbf5554a2faa5/app/src/main/java/com/example/mytrainingpal/util/Utils.kt#L15

Reply:
Surio